### PR TITLE
Remove React

### PR DESCRIPTION
**Pull request by dantailby**

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -0,0 +1 @@
+module.exports = require('morph-shovel').load('sport-icons',  STATIC_ASSET_PATH + '/gel-icons-sport-set.svg');
```
Comment (mintuz) [inline index.js]: more curious than anything. does STATIC_ASSET_PATH come from Morph?

File: test/index-spec.js [hunk 2]
```diff
@@ -0,0 +1,9 @@
+xdescribe('Morph Shovel Sport', function() {
+    
+    // setup code
+    
+    it('Should enable sport icons', function() {
+        // test code
```
Comment (mintuz) [inline test/index-spec.js]: tests are specific :trollface:

File: index.jsx [hunk 3]
```diff
@@ -1,27 +0,0 @@
...
-
-    console.log(Shovel);
-
-    if(Morph.isClientSide()) {
-      Shovel.load('sport-icons', '/dist/gel-icons-sport-set.svg');
-      window.alert('hey');
-    }
-
-    return (
-        <h2>Bulmer</h2>
```
Comment (mintuz) [inline index.jsx]: hello cheeky.

File: README.md [hunk 4]
```diff
@@ -0,0 +1,2 @@
+# Morph Shovel Sport
+Enables use of Sport Icon set from [GEL Iconography](http://bbc.github.io/gel-iconography/) in any Morph component using [Shovel](https://github.com/bbc/shovel).
```
Comment (mintuz) [inline README.md]: can't comment on it for some reason but do we need the dist folder checked in.

File: index.js (hunk 1 above)
Comment (dantailby) [inline index.js]: The pack process will replace `STATIC_ASSET_PATH` with the correct path to the file

Comment (mintuz) [inline index.js]: :+1:

**Pull request merged**